src/cocaine/burlak/burlak.py

```diff
@@ -837,6 +839,8 @@ def process_loop(self):
         control_filter = yield self.get_filter_once()
         no_state_yet = True
 
+        run_lock = LockHolder()
```

Comment:
Could `LockHolder` wrapper be dropped? And use underlying lock as is:
```python
run_lock = None
# ...
run_lock = yield semaphore.release_lock_holder(run_lock)
# ...
if run_lock is None:
    # ...
    run_lock = yield semaphore.try_to_acquire_lock()
# ...
```

Reply:
No, it is needed as keeper of object, which would be reseted to None from another class to signal `start_app` completion.